contracts/nft/mainnet/NFTMainnetBridgingToPolygonFacet.sol

```diff
@@ -120,10 +120,37 @@ contract NFTMainnetBridgingToPolygonFacet {
         __depositFor(tokenIds_, amounts_);
     }
 
-    function bridgeNFTsV2(
-        uint256[] calldata tokenIds,
-        uint256[] calldata amounts
-    ) external {
-        // TODO: waiting for code Syed is working on
+    /**
+     * @notice Gets all of our NFTs (staked and unstaked) migrates to ERC1155 (if ERC721), then
+     *  bridges to polygon
+     * @param tokenIds The tokenIds that we are sending. First array is the ERC721, if any. Second array
+     *  are our ERC-1155 tokenIDs
+     */
+
+    function bridgeNFTsV2(uint256[] calldata tokenIds) external {
```

Comment:
Looks like the `amounts` parameter was removed. This should stay as a parameter passed in rather than fetching the `balanceOf` below on line 140